lib/resources/example_json/soarian_examples_conformance.rb

```diff
@@ -0,0 +1,323 @@
+    "kind": "capability",
+    "software": {
+        "name": "Soarian Clinicals CCDS API",
+        "version": "01.00.0000"
+    },
+    "fhirVersion": "HAPI-1.3, FHIR DSTU2 - 1.0.2",
+    "acceptUnknown": "extensions",
+    "format": [
+        "application/xml+fhir",
+        "application/json+fhir"
+    ],
+    "rest": [
+        {
+            "mode": "server",
+            "security": {
+                "extension": [
+                    {
+                        "url": "http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris",
+                        "extension": [
+                            {
+                                "url": "token",
+                                "valueUri": "https://authorization.sandboxcerner.com/tenants/45343c0c-3f93-4721-874a-258fc4bae3a4/protocols/oauth2/profiles/smart-v1/token"
+                            },
+                            {
+                                "url": "authorize",
+                                "valueUri": "https://authorization.sandboxcerner.com/tenants/45343c0c-3f93-4721-874a-258fc4bae3a4/protocols/oauth2/profiles/smart-v1/personas/patient/authorize"
+                            }
+                        ]
+                    },
+                    {
+                        "url": "http://fhir-registry.smarthealthit.org/StructureDefinition/SoarianIdLinking-uris",
+                        "extension": [
+                            {
+                                "url": "token",
+                                "valueUri": "https://authorization.sandboxcerner.com/tenants/45343c0c-3f93-4721-874a-258fc4bae3a4/protocols/oauth2/profiles/soarian-identity-linking-v1/token"
+                            },
+                            {
+                                "url": "authorize",
+                                "valueUri": "https://authorization.sandboxcerner.com/tenants/45343c0c-3f93-4721-874a-258fc4bae3a4/protocols/oauth2/profiles/soarian-identity-linking-v1/personas/patient/authorize"
+                            }
+                        ]
+                    }
+                ],
+                "description": "OAuth2 with SMART extensions"
+            },
+            "resource": [
+                {
+                    "type": "AllergyIntolerance",
+                    "profile": {
+                        "reference": "http://hl7.org/fhir/profiles/AllergyIntolerance"
+                    },
+                    "interaction": [
+                        {
+                            "code": "search-type"
+                        }
+                    ],
+                    "searchParam": [
+                        {
+                            "name": "patientId",
+                            "type": "string",
+                            "documentation": "https://fhir-myrecord.sandboxcerner.com/dstu2/45343c0c-3f93-4721-874a-258fc4bae3a4/swagger/swagger.yaml"
```

Comment:
Is this publicly accessible and should this always be used by an app wanting to view doc?